workshops/migrations/0087_auto_20160519_0739.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.6 on 2016-05-19 12:39
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+import requests
+
+
+def populate_languages(apps, schema_editor,
+        url='https://raw.githubusercontent.com/mattcg/language-subtag-registry/master/data/json/registry.json'):
```

Comment:
Please cache a copy of this locally - we have no control over external resources (they can be renamed or deleted without warning), and I'd like to be able to run tests offline (I spend more time than I'd like on planes).